runtime/lua/vim/lsp/handlers.lua

```diff
@@ -644,6 +644,10 @@ M[ms.workspace_inlayHint_refresh] = function(err, result, ctx, config)
   return vim.lsp.inlay_hint.on_refresh(err, result, ctx, config)
 end
 
+M[ms.workspace_semanticTokens_refresh] = function(err, result, ctx, _config)
```

Comment:
Just for the sake of consistency, it'd be great to move the URL link as well:

```suggestion
---@see https://microsoft.github.io/language-server-protocol/specifications/specification-current/#semanticTokens_refreshRequest
M[ms.workspace_semanticTokens_refresh] = function(err, result, ctx, _config)
```